tests/s3/versioning/test_get_put_object_tagging.py

```diff
@@ -153,7 +153,7 @@ def test_get_put_obj_tags_40429(self):
                                              object_name=self.object_name,
                                              version_tag=self.ver_tag, versions_dict=self.versions)
         assert_utils.assert_true(resp[0], resp)
-
+        put_tag = self.ver_tag[self.object_name][latest_v][-1]
```

Comment:
Is this required?

Reply:
if we are puting new tag key-val, we supposed to fetch the same before comparing it with GET right ?

Reply:
Did you try this test after removing this put_tag?

Reply:
This has bin tested, when this new fetch was not there test was failing, as in step 6 above this line we are again doing put tag. Hence we need to fetch the new one before comparing it with new get tag